tests/Filter/QueryBuilder.php

```diff
@@ -48,20 +48,18 @@ public function expr()
     }
 
     /**
-     * @param string $name
-     * @param string $value
+     * @param string          $alias
```

Comment:
```suggestion
     * @param string $alias
```

Reply:
Why ? There is no lint error, and I saw a lot of function which align the params.
![image](https://user-images.githubusercontent.com/9052536/77085946-5746f100-6a01-11ea-9f3a-0150d6c10664.png)

Reply:
I know, we haven't configured the CS rule `phpdoc_align`. My suggestion is just based on my own experience dealing with conflicts in other lines when some argument is renamed, added, removed or replaced.
Just ignore the suggestion if you don't agree :+1: